ethereum/pow/merkletree/src/main/java/tech/pegasys/teku/ethereum/pow/merkletree/DepositTreeSnapshot.java

```diff
@@ -31,6 +32,13 @@ public DepositTreeSnapshot(
     this.executionBlockHash = executionBlockHash;
   }
 
+  public DepositTreeSnapshot(
+      final tech.pegasys.teku.spec.datastructures.eth1.DepositTreeSnapshot depositTreeSnapshot) {
```

Comment:
Ultimately we're going to want to be able to start from just a `tech.pegasys.teku.spec.datastructures.eth1.DepositTreeSnapshot` because we want to allow users to provide one of those and save us loading all deposits from the eth1 chain.  We can get the eth1 block number from the `executionBlockHash` included in the deposit snapshot. That will require a bunch more changes to how we start up the PowChainService and tell it where to start fetching deposits from - prior to this it's always just been given a block number, but it now needs to take a block hash and get the number from that. It may also have to deal with that block hash being unknown if the eth1 node changed and isn't in sync now too.